corehq/apps/repeaters/models.py

```diff
@@ -284,14 +285,17 @@ def get_auth(self):
             return HTTPDigestAuth(self.username, self.password)
         return None
 
+    def send_request(self, payload, url, headers, auth):
+        return simple_post(payload, url, headers=headers, timeout=POST_TIMEOUT, auth=auth)
+
     def fire_for_record(self, repeat_record):
         headers = self.get_headers(repeat_record)
         auth = self.get_auth()
         payload = self.get_payload(repeat_record)
         url = self.get_url(repeat_record)
 
         try:
-            response = simple_post(payload, url, headers=headers, timeout=POST_TIMEOUT, auth=auth)
+            response = self.send_request(payload, url, headers, auth)
```

Comment:
I don't have a preference to either of those approaches as of now. Just did the one which was less code changes. Making that change if that sounds better.